nntrainer/include/tensor_dim.h

```diff
@@ -45,6 +45,8 @@ class TensorDim {
 
   TensorDim(const TensorDim &rhs) = default;
 
+  TensorDim(const std::string &shape) : TensorDim() { setTensorDim(shape); }
```

Comment:
setTensorDim can fail with error return code.
please check that.

Reply:
Done